storage.lua

```diff
@@ -1,17 +1,10 @@
-
 function mail.getMailFile(playername)
 	local saneplayername = string.gsub(playername, "[.|/]", "")
 	return mail.maildir .. "/" .. saneplayername .. ".json"
 end
 
-function mail.getContactsFile(playername)
-	local saneplayername = string.gsub(playername, "[.|/]", "")
-	return mail.maildir .. "/contacts/" .. saneplayername .. ".json"
-end
-
-
-function mail.getMessages(playername)
-	local messages = mail.read_json_file(mail.getMailFile(playername))
+function mail.getMessages()
```

Comment:
Consider this for better backwards compatibility (if returned data is still backwards compatible):
```suggestion
function mail.getMessages(playername)
	if (playername) then
		return mail.getPlayerMessages(playername)
	end
```